lib/single-click-open.js

```diff
@@ -25,10 +25,10 @@ export function activate() {
   disposables = new CompositeDisposable();
 
   if (atom.config.get('single-click-open.fixTreeView'))
-    disposables.add(disablePendingItems());
+    disablePendingItems();
 
   if (atom.config.get('single-click-open.fixSearchResultsView'))
-    disposables.add(disableSearchResultDoubleClick());
+    disableSearchResultDoubleClick();
 }
 
 export function deactivate() {
```

Comment:
in `deactivate()` it calls `disposable.dispose()` and it should be `disposables`